src/documents/tests/test_bulk_edit.py

```diff
@@ -275,3 +278,266 @@ def test_set_permissions_merge(self, m):
             self.doc1,
         )
         self.assertEqual(groups_with_perms.count(), 2)
+
+
+class TestPDFActions(DirectoriesMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+        sample1 = os.path.join(self.dirs.scratch_dir, "sample.pdf")
```

Comment:
Sorry, it's 100% a nit, but this should be possible as a Path operation as well.  I believe I converted the utility to output Path types

Reply:
ha no worries, lmk if that does it.